### PR TITLE
feat: add write_aspect_bazelrc_presets macro

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

**Review by alexeagle — APPROVED** [review]

**Comment by alexeagle** [comment]
Will this show properly on the docsite?

**Comment by gregmagolan** [comment]
> 

I like preset. Line up better with existing patterns. I'll update.

**Comment by gregmagolan** [comment]
> Will this show properly on the docsite?

Good point. I could make this README the SoT for the docsite itself.

File: .aspect/bazelrc/README.md [hunk 1]
```diff
@@ -1,4 +1,73 @@
...
+# this file. See https://bazel.build/configure/best-practices#bazelrc-file.
+try-import %workspace%/.aspect/bazelrc/user.bazelrc
+```
+
+## Automatic updates
+
+A convenient way to automatically keep your vendored copy up-to-date is to use
+[`write_source_files`](https://docs.aspect.build/rules/aspect_bazel_lib/docs/write_source_files):
+
+```
```
Comment (alexeagle) [inline .aspect/bazelrc/README.md:46]: say that this is the content of `.aspect/bazelrc/BUILD.bazel`

File: .aspect/bazelrc/README.md [hunk 2]
```diff
@@ -1,4 +1,73 @@
...
+load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_files")
+
+# Keep up-to-date with Aspect recommended .bazelrc settings.
+# See https://github.com/aspect-build/bazel-lib/tree/main/.aspect/bazelrc for the most up to date
+# version of Aspect's recommended Bazel options. These are mirrored on our
+# [docsite](https://docs.aspect.build/guides/bazelrc) for easier consumption.
+write_source_files(
+    name = "update_aspect_recommended_bazel_options",
+    files = {
+        "bazel5.bazelrc": "@aspect_bazel_lib//.aspect/bazelrc:bazel5.bazelrc",
```
Comment (alexeagle) [inline .aspect/bazelrc/README.md:56]: maybe bazel-lib should expose a macro that includes all of these?

File: .aspect/bazelrc/README.md [hunk 3]
```diff
@@ -1,4 +1,73 @@
...
 They are mirrored on our docsite at https://docs.aspect.build/guides/bazelrc.
+
+## Using these Bazel options in your project
+
+To use these files in your project, simply vendor them in and reference them from your `.bazelrc` file.
+
+For example,
+
+```
+# Import Aspect recommended Bazel convenience settings for all projects
```
Comment (alexeagle) [inline .aspect/bazelrc/README.md:14]: nit: it's a lot of extra lines for these repeated comments, could just be one comment and then 8 import lines

Comment (gregmagolan) [reply .aspect/bazelrc/README.md:14]: that was bothering me as well; I'll update

File: .aspect/bazelrc/README.md (hunk 1 above)
Comment (gregmagolan) [reply .aspect/bazelrc/README.md:35]: 👍

File: .aspect/bazelrc/README.md (hunk 2 above)
Comment (gregmagolan) [reply .aspect/bazelrc/README.md]: ✅

**Pull request merged**